### ISSUE TITLE
How to use with delve

### ISSUE THREAD

**Issue by jesseduffield** [gotestyourself/gotestsum#229]
I'm not sure if I'm using this package with delve properly. Here's my current flow

1) run `gotestsum --watch`
2) press 'd' to have the test run with delve
3) work through the breakpoints (e.g. by just pressing 'c')
4) observe that test fails
5) delve prints `Process 51458 has exited with status 1`
6) I get to the dlv prompt again but I'm not sure how to actually restart the test now without exiting and running `gotestsum --watch` again.

Is there something I need to run to get the test to restart without killing the session?

Thanks!

**Comment by jesseduffield** [comment]
Ah sorry I now see that I need to press 'r' to restart the session. I'll close this off. Might be useful to add some documentation about this flow in the readme for those unfamiliar with delve.

**Comment by dnephin** [comment]
Good call! I think I was confused about this the first time I ran it as well. I'll re-open so that others can find this issue, until it makes it into the docs.

If I remember correctly, I thought I was using either `ctrl-D` or the `exit` command to exit delve after the test ran, but it's been a while, so I may be remembering wrong.
